scripts/diff.js

```diff
@@ -0,0 +1,122 @@
+#!/usr/bin/env node
+'use strict'
+
+const spawn = require('child_process').spawn
+const fs = require('fs/promises')
+const path = require('path')
+const util = require('util')
+const exec = util.promisify(require('child_process').exec)
+require('make-promises-safe') // exit on error
+
+const DIST_ROOT = '/ipns/dist.ipfs.io'
```

Comment:
#367  will need this for consistency during some custom runs:

```suggestion
const DIST_ROOT = process.env.DIST_ROOT || '/ipns/dist.ipfs.io'
```